doc/source/guide/geometry_guide.rst

```diff
@@ -0,0 +1,240 @@
+
+where :math:`\Omega \subset \mathbb{R}^2` is a bounded domain, :math:`\mathbb{S}^1` is the unit sphere in 2 dimensions and :math:`\theta^\perp = \{x \in \mathbb{R}^2\, |\, \langle x,\, \theta \rangle = 0\}` is the plane (=line) perpendicular to a vector :math:`\theta`.
+In this case, the data manifold is
+
+    .. math::
+        M = \big\{(\theta, v)\,\big|\, \theta \in \Gamma,\ v \in \theta^\perp \big\}
+
+and encodes the subset of lines in :math:`\mathbb{R}^2` that are parallel to a unit vector in :math:`\Gamma`.
+
+
+Representation using Euclidean domains
+--------------------------------------
+Function spaces (discrete or continuous) in ODL are, up to a few exceptions, defined on rectangular domains.
+Such spaces have a relatively simple structure and can be represented and manipulated very efficiently.
+Therefore ODL does not represent data directly as functions on manifolds, but rather as functions on a *coordinate domain* that paremetrizes the manifold.
+
+For instance, in the 2D parallel beam example above, a unit vector :math:`\theta \in \mathbb{S}^1` can be parametrized by an angle :math:`\varphi \in [0, 2\pi)`, and a vector :math:`v` on the line :math:`\theta^\perp` by a single number :math:`s \in \mathbb{R}`.
+Such a representation additionally requires a *convention* for a translation between coordinates :math:`(\varphi, s)` and points :math:`(\theta, v)` on the manifold, i.e., a map between the coordinate domain and the data manifold.
+Such a map is usually called a *parametrization* or `chart`_ of the manifold.
+
+In our example, we could thus redefine the ray transform as a map between Euclidean function spaces:
+
+    .. math::
+        & \mathcal{R} :  L^2(\Omega) \to L^2(I \times D),
+
+        & \mathcal{R}(f)(\varphi, u) := \int_{\mathbb{R}} f\big( t\theta(\varphi) + u\theta(\varphi + \pi/2) \big)\, \mathrm{d}t.
+
+Here, :math:`I \subset [0, 2\pi)` and :math:`D \subset \mathbb{R}` are intervals and
+
+    .. math::
+        & (\varphi, u) \mapsto \big( \theta(\varphi), u \theta(\varphi + \pi/2)\big),
+
+        & \theta(\varphi) := (\cos\varphi, \sin\varphi)
+
+is the parametrization of the data manifold.
+
+
+.. _manifold: https://en.wikipedia.org/wiki/Manifold
+.. _chart: https://en.wikipedia.org/wiki/Manifold#Charts
+
+
+
+Geometries in ODL
+=================
+The `RayTransform` in ODL is an `Operator` between discrete function spaces defined on rectangular domains.
+These rectangular regions either stand for a reconstruction region ("volume") or for an interval product of parameters describing the data.
+The missing component, which is the translation of coordinates to points on the data manifold, is encoded in the `Geometry` class and its subclasses as described in the following.
+
+
+The `Geometry` class
+--------------------
+All ODL geometries derive from the abstract `Geometry` class and need to call its ``__init__`` method to populate the basic structure.
```

Comment:
If I leave out the `__init__` part, this is a high-level description of what all geometries have in common. Maybe that already solves the issue.